Description: Fix use FHS-compatible state dir /var/lib/netatalk
Author: Jonas Smedegaard <dr@jones.dk>
Forwarded: no
Last-Update: 2019-02-15

Modified for Alpine

---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
diff --git a/doc/manual/configuration.xml b/doc/manual/configuration.xml
index 96d3947..ee6436a 100644
--- a/doc/manual/configuration.xml
+++ b/doc/manual/configuration.xml
@@ -173,7 +173,7 @@ basedir regex = /usr/home</programlisting></para>
       database storing ID &lt;-&gt; name mappings.</para>
 
       <para>The CNID databases are by default located in
-      <filename>/var/netatalk/CNID</filename>. You can pass
+      <filename>/var/lib/netatalk/CNID</filename>. You can pass
       <command>--localstatedir=PATH</command> to the configure script
       to change the location.</para>
 
@@ -1622,9 +1622,9 @@ aclmode = passthrough</screen>
       <screen>$ su
       # cat .tracker_profile
       PREFIX="/usr/local"
-      export XDG_DATA_HOME="$PREFIX/var/netatalk/"
-      export XDG_CACHE_HOME="$PREFIX/var/netatalk/"
-      export DBUS_SESSION_BUS_ADDRESS="unix:path=$PREFIX/var/netatalk/spotlight.ipc"
+      export XDG_DATA_HOME="$PREFIX/var/lib/netatalk/"
+      export XDG_CACHE_HOME="$PREFIX/var/lib/netatalk/"
+      export DBUS_SESSION_BUS_ADDRESS="unix:path=$PREFIX/var/lib/netatalk/spotlight.ipc"
       # . .tracker_profile
       #
       </screen></para>
diff --git a/doc/manual/upgrade.xml b/doc/manual/upgrade.xml
index bbe03ca..caae02d 100644
--- a/doc/manual/upgrade.xml
+++ b/doc/manual/upgrade.xml
@@ -137,8 +137,8 @@
 
           <listitem>
             <para>The CNID databases are now stored under
-              <filename>/var/netatalk/CNID/</filename>
-              by default. You can use <command>configure --localstatedir=PATH</command>
+              <filename>/var/lib/netatalk/CNID/</filename>
+              on Alpine systems. You can use <command>configure --localstatedir=PATH</command>
               at compile time to change the location.</para>
           </listitem>
 
@@ -183,8 +183,8 @@
 
         <listitem>
           <para>Move <filename>afp_voluuid.conf</filename> and
-          <filename>afp_signature.conf</filename> to the localstate directory (default
-          <filename>/var/netatalk/</filename>), you can use <command>afpd -v</command>
+          <filename>afp_signature.conf</filename> to the localstate directory (Debian uses
+          <filename>/var/lib/netatalk/</filename>), you can use <command>afpd -v</command>
           in order to find the correct path</para>
         </listitem>
 
